curvefs/src/client/s3/client_s3_adaptor.cpp

```diff
@@ -113,30 +112,27 @@ int S3ClientAdaptorImpl::Write(uint64_t inodeId, uint64_t offset,
     VLOG(6) << "write start offset:" << offset << ", len:" << length
             << ", fsId:" << fsId_ << ", inodeId:" << inodeId;
     uint64_t start = butil::cpuwide_time_us();
-    FileCacheManagerPtr fileCacheManager =
-        fsCacheManager_->FindOrCreateFileCacheManager(fsId_, inodeId);
     {
-        std::lock_guard<std::mutex> lockguard(ioMtx_);
+        std::lock_guard<std::mutex> lockGuard(ioMtx_);
         pendingReq_.fetch_add(1, std::memory_order_seq_cst);
         VLOG(6) << "pendingReq_ is: " << pendingReq_;
-        uint64_t pendingReq = pendingReq_.load(std::memory_order_seq_cst);
         fsCacheManager_->DataCacheByteInc(length);
         uint64_t size = fsCacheManager_->GetDataCacheSize();
-        uint64_t maxSize = fsCacheManager_->GetDataCacheMaxSize();
-        if ((size + pendingReq * fuseMaxSize_) >= maxSize) {
+        const uint64_t maxSize = fsCacheManager_->GetDataCacheMaxSize();
+        if (size >= maxSize) {
```

Comment:
pendingReq_ can also be deleted, and i think you maybe use another pr to remove fusemax and pendingReq,it will be clearer(this pr just edit cache size only)

Reply:
Good suggestion1! Let me rearrange them.